experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1919,6 +1920,41 @@ describe("SharedTree", () => {
 			assert(compareUpPaths(expectedPath, anchorPath));
 		});
 	});
+
+	describe("Creates a SharedTree using specific ForestType", () => {
+		it("unspecified ForestType uses ObjectForest", () => {
+			const { trees } = new TestTreeProviderLite(
+				1,
+				new SharedTreeFactory({
+					jsonValidator: typeboxValidator,
+					forest: ForestType.Reference,
+				}),
+			);
+			assert.equal(trees[0].forest.computationName, "object-forest.ObjectForest");
```

Comment:
I didn't realize that would be a way to do this test. Neat trick, and keeps the test simple.

Reply:
:)